addons/mass_mailing/tests/test_mailing_internals.py

```diff
@@ -193,6 +193,27 @@ def test_mailing_computed_fields_form(self):
         )
         self.assertEqual(mailing_form.mailing_model_real, 'res.partner')
 
+    def test_mass_mailing_values_for_contacts(self):
```

Comment:
Would test with a test user that is a marketing user, just to try out ACLs. It should be available as test data (marketing_user would say).